src/PendingReason.php

```diff
@@ -54,10 +54,58 @@ public static function getTypeName($nb = 0)
 
     public function getAdditionalFields()
     {
+        $defaultPendingReason = self::getDefault();
+
         return [
+            [
+                'name' => 'is_default',
+                'label' => __('Default pending reason'),
+                'type' => 'bool',
+                'params' => [
+                    'add_field_html' => ($defaultPendingReason && $defaultPendingReason->getID() != $this->getID() ?
```

Comment:
To avoid writing too much HTML/javascript here as a long concatenated string, maybe we could create a twig template for this data and returns its content.